data/styles/arrow.css

```diff
@@ -26,7 +26,7 @@
         0 1px 2px rgba(0,0,0,0.24);
     margin: 6px;
     padding: 6px;
-    color: #333;
+    color: @textColorPrimary;
```

Comment:
You don’t want to use textcolorprimary here because it mixes with the title bar color. Instead use `@text_color`

Reply:
ah derp